src/Uno.UWP/Storage/FileIO.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Globalization;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Uno.Extensions;
+using Windows.Foundation;
+using Windows.Storage.FileProperties;
+using Windows.Storage.Streams;
+
+namespace Windows.Storage
+{
+	public partial class FileIO
+	{
+
+		private static async Task AppendWriteTextAsync(global::Windows.Storage.IStorageFile file, string contents, global::Windows.Storage.Streams.UnicodeEncoding encoding, bool append)
+		{
+			if (file is null)
+			{
+				throw new NullReferenceException("StorageFile cannot be null");
+			}
+
+			global::System.Text.Encoding encodingForWriter = global::System.Text.Encoding.UTF8;
```

Comment:
Please use namespaces to avoid fully qualified names in non-generated files.

Reply:
Done. But I left one level up in Encoding, as we have here two different encodings.